spec/requests/providers/capital_income_assessment_results_controller_spec.rb

```diff
@@ -38,14 +38,36 @@
         end
 
         context "when not eligible" do
-          let(:cfe_result) { create(:cfe_v4_result, :not_eligible) }
+          context "when ineligible and gross income is above the upper threshold" do
+            let(:cfe_result) { create(:cfe_v6_result, :ineligible_gross_income) }
 
-          it "returns http success" do
-            expect(response).to have_http_status(:ok)
+            it "returns http success" do
+              expect(response).to have_http_status(:ok)
+            end
+
+            it "displays the correct result" do
+              expect(unescaped_response_body).to include(I18n.t("ineligible.heading", name: applicant_name, scope: locale_scope))
+            end
+
+            it "displays the correct reason" do
+              expect(unescaped_response_body).to include("This is because they have too much gross income")
+            end
           end
 
-          it "displays the correct result" do
-            expect(unescaped_response_body).to include(I18n.t("ineligible.heading", name: applicant_name, scope: locale_scope))
+          context "when ineligible and disposable income is above the upper threshold" do
+            let(:cfe_result) { create(:cfe_v6_result, :ineligible_disposable_income) }
+
+            it "returns http success" do
+              expect(response).to have_http_status(:ok)
+            end
+
+            it "displays the correct result" do
+              expect(unescaped_response_body).to include(I18n.t("ineligible.heading", name: applicant_name, scope: locale_scope))
```

Comment:
**nit** I think it is clear and simpler and less coupled to simple expect the text/string that you expect

Reply:
👍 have amended to use string instead of translation in spec expectation